servicecontrol/monitoring-instances/installation/creating-config-file.md

```diff
@@ -110,3 +102,18 @@ Type: timespan
 
 Default: 40 seconds
 
+## Usage Reporting
+
+### Monitoring/ServiceControlThroughputDataQueue
+
+The queue on which throughput data is received by ServiceControl Error instance. This setting must match the equivalent [`LicensingComponent/ServiceControlThroughputDataQueue`](/servicecontrol/creating-config-file.md#usage-reporting-licensingcomponentservicecontrolthroughputdataqueue) setting on the ServiceControl Error instance.
+
+In most instances these settings do not need to be modified.
+
+If running multiple setups of the Platform Tools (i.e. multiple versions of ServiceControl Error and Monitoring instances) then modify these settings so that the queue on each Monitoring instance is matched to the queue of its Error instance.
```

Comment:
```suggestion
If running multiple setups of the Platform Tools (i.e. multiple versions of ServiceControl Error and monitoring instances) then modify these settings so that the queue on each Monitoring instance is matched to the queue of its error instance.
```